lib/koans/10_processes.ex

```diff
@@ -2,17 +2,17 @@ defmodule Processes do
   use Koans
 
   koan "Tests run in a process" do
-    assert Process.alive?(self()) == ___
+    assert Process.alive?(self) == ___
```

Comment:
I tend to go with Avdi's [barewords](http://devblog.avdi.org/2012/10/01/barewords/) semantic for this kind of thing. Maybe not as applicable to Elixir code as it is to Ruby, but it's my history 😄